core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1436,17 +1438,15 @@ class ReplicaManager(val config: KafkaConfig,
 
           leaderAndIsrRequest.partitionStates.forEach { partitionState =>
             val topicPartition = new TopicPartition(partitionState.topicName, partitionState.partitionIndex)
-            /*
-           * If there is offline log directory, a Partition object may have been created by getOrCreatePartition()
-           * before getOrCreateReplica() failed to create local replica due to KafkaStorageException.
-           * In this case ReplicaManager.allPartitions will map this topic-partition to an empty Partition object.
-           * we need to map this topic-partition to OfflinePartition instead.
-           */
+            // If there is offline log directory, a Partition object may have been created by getOrCreatePartition()
+            // before getOrCreateReplica() failed to create local replica due to KafkaStorageException.
```

Comment:
Hmm, `getOrCreateReplica` no longer exists. Maybe `createLogIfNotExists` is the replacement?